### DISCUSSION TITLE
Creating a utility function that can help find if an extension exists in a particular Questionnaire.Item

### DISCUSSION THREAD

**Discussion by kunjan8794** [google/android-fhir#200] [Ideas]
We can create a utility function that can be useful for finding if the Questionnaire.Item has a particular extension.

It can go like a simple function for example, named --> hasExtension(extensionName) which can return a boolean value showing whether the extension exists. 

This can significantly reduce redundant code for finding extensions' existence(!) maybe that would be written for every possible validation check

@jingtang10 @joiskash

**Comment by kunjan8794** [comment]
@jingtang10 has suggested that the return value can be the extension itself or null - that's better it seems

**Comment by jingtang10** [comment]
great idea!

might be worth looking into [kotlin extension function](https://kotlinlang.org/docs/reference/extensions.html)

**Reply by kunjan8794** [reply]
@jingtang10  Looking at the link, it seems maybe we can create an extension function for the class Questionnaire.Item ?
